brickschema/brickify/src/handlers/Handler/HaystackHandler/utils/HaystackRDFInferenceSession.py

```diff
@@ -0,0 +1,62 @@
+import rdflib
+from brickschema.inference import HaystackInferenceSession
+from brickschema.namespaces import BRICK, A, RDFS
+from rdflib import Namespace, Graph
+from typer import progressbar
+
+
+class HaystackRDFInferenceSession(HaystackInferenceSession):
+    def __init__(self, namespace):
+        super().__init__(namespace)
+        self._BLDG = Namespace(namespace)
+        self._tagmap = {
+            "cmd": "command",
+            "sp": "setpoint",
+            "temp": "temperature",
+            "lights": "lighting",
+            "rtu": "RTU",
+            "ahu": "AHU",
+            "freq": "frequency",
+            "equip": "equipment",
```

Comment:
Isn't there already an abbreviation map in `brickschema`? Are there mappings that are missing in my implementation? Any way we can bring them together?